code/modules/lighting/lighting_object.dm

```diff
@@ -90,6 +94,10 @@
 	var/set_luminosity = max > 1e-6
 	#endif
 
+	// Respect daynight blending from an area for luminosity here.
```

Comment:
what happens if you do not?

Reply:
If not, this will cause the turf to not respect the adjacent lighting, e.g. if it's day outside and this turf gets its luminosity updated to be dark, it will not respect the outside light and then it would become dark and only update when the outside lighting updates.

Reply:
can you add that to the comment please

Reply:
Sure